### PR TITLE
Instance size fix

### LINKED ISSUE #17: 3.x and 4.x : Use smaller EC2 instances
Current instance configurations : 

OCP 3 : 
- infranode x 1 (m4.4xlarge)
- node x 2 (m4.4xlarge)
- master x 1 (m4.4xlarge)
- support x 1 (m4.4xlarge)
- bastion x 1 (t2.large)

OCP 4 : 
- master x 3 (m4.xlarge)
- worker x 3 (m4.4xlarge)
- bootstrap x 1 (m4.large)

Investigate if we can use smaller instances. 

For OCP 4, investigate if three masters are required.

### PR DESCRIPTION
**Pull request by pranavgaikwad**
Fixes #17

### CODE REVIEW

File: 4.x/ocp4_vars.yml [hunk 1]
```diff
@@ -25,3 +25,10 @@ student_workloads: []
 # discouraged to do so.
 # default_workloads: [ocp4-workload-enable-lets-encrypt-certificates]
 
+clientvm_instance_type: "t2.medium"
+clientvm_instance_count: 1
+master_instance_type: "m5.xlarge"
+master_instance_count: 3
```
Comment (pranavgaikwad) [inline 4.x/ocp4_vars.yml]: according to OpenShift docs, 3 masters (xlarge+) and 2 compute (large+) is a minimum requirement.

Comment (jwmatthews) [reply 4.x/ocp4_vars.yml]: The recommendation may be for a production cluster.

I was hoping we could run with less masters for dev clusters.
In past we would run 1 Master and 1 (or 2) worker nodes.


How about we try with 1 master, see how it behaves.  If we see issues we bump to more masters?

Comment (pranavgaikwad) [reply 4.x/ocp4_vars.yml]: I tried running with one master. But my installation did not proceed after 85%. the openshift installer then timed out. I can try again.

**Comment by jwmatthews** [comment]
I tested this branch and had success.

Let's merge it as-is, and we can come back to investigate the # of masters.
In prior versions of openshift-install we would run with 1 master, it's possible something changed with 4.1.0 and more are needed.  I was trying recently with 1 master for a few runs and seeing intermittent issues...maybe using 1 master was causing a problem....I changed to your branch and both clusters came up.

**Pull request merged**